src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Completion/DefaultTagHelperCompletionService.cs

```diff
@@ -221,50 +214,13 @@ public override IReadOnlyList<CompletionItem> GetCompletionsAt(SourceSpan locati
             return completionItems;
         }
 
-        private bool TryResolveAttributeInsertionSnippet(
-            string text,
-            IEnumerable<BoundAttributeDescriptor> boundAttributes,
-            bool indexerCompletion,
-            out string snippetText)
-        {
-            var snippetSupported = LanguageServer.ClientSettings?.Capabilities?.TextDocument?.Completion.Value?.CompletionItem?.SnippetSupport ?? false;
```

Comment:
I think you can remove this from being injected into this class now

Reply:
Good catch, removed!